schema/v1alpha1/sensor.proto

```
@@ -105,68 +188,47 @@ message DiscreteSensorReading {
+  // Cross-field constraint for min/max values
+  option (buf.validate.message).cel = {
+    id : "min_max_values",
+    message : "min_value must be less than or equal to max_value",
+    expression : "this.min_value <= this.max_value"
+  };
+
+  option (buf.validate.message).cel = {
+    id : "min_max_timestamps",
+    message : "timestamps must be valid",
+    expression : "this.min_timestamp > 0 && this.max_timestamp > 0"
+  };
```

Buf (1.55.1)

(buf.validate.message).cel.expression on message "MinMaxRecorded" fails to compile: found no matching overload for '_>_' applied to '(timestamp, int)'
 | this.min_timestamp > 0 && this.max_timestamp > 0
 | .............................................^

(PROTOVALIDATE)